internal/controllers/topology/cluster/scope/upgradetracker.go

```diff
@@ -77,12 +78,21 @@ func (m *MachineDeploymentUpgradeTracker) RolloutNames() []string {
 	return m.rollingOutNames.List()
 }
 
+// HoldUpgrades is used to set if any subsequent upgrade operations should be paused.
```

Comment:
```suggestion
// HoldUpgrades is used to set if any subsequent upgrade operations should be paused,
// e.g. because a beforeUpgrade hook response asked to do so.
```